controllers/pokemon_controller.go

```diff
@@ -0,0 +1,19 @@
+package controllers
```

Comment:
Remember to name your packages in singular

Reply:
Also, you don't need to add the suffix `controller` to your files, the package gives enough context to tell what they are